modules/apps/commerce/commerce-initializer-util/src/main/java/com/liferay/commerce/initializer/util/internal/osgi/commands/CommerceOSGiCommands.java

```diff
@@ -60,9 +58,9 @@ public void initializeSite(long groupId, String key) throws Exception {
 		Role role = _roleLocalService.fetchRole(
 			company.getCompanyId(), RoleConstants.ADMINISTRATOR);
 
-		List<User> roleUsers = _userLocalService.getRoleUsers(role.getRoleId());
+		long[] userIds = _userLocalService.getRoleUserIds(role.getRoleId());
 
-		User user = roleUsers.get(0);
+		User user = _userLocalService.getUser(userIds[0]);
```

Comment:
@marianoalvarosaiz This does not make sense to me.

Before, there is only invocation to _userLocalService.
After, there are two invocations to _userLocalService.

How do you know "after" is performing better than "before"?